cypress/integration/tests/controls/stakeholders/create.test.ts

```diff
@@ -0,0 +1,120 @@
+    });
+
+    it("Stakeholder field validations", function () {
+        // Navigate to stakeholder tab and click create new button
+        clickByText(navMenu, controls);
+        clickByText(navTab, stakeholders);
+        clickByText(button, createNewButton);
+
+        // Check "Create" and "Cancel" button status
+        cy.get(commonView.submitButton).should("be.disabled");
+        cy.get(commonView.cancelButton).should("not.be.disabled");
+
+        // Email constraints
+        inputText(stakeholderEmailInput, data.getRandomWord(2));
+        cy.get(emailHelper).should("contain", minCharsMsg);
+        inputText(stakeholderEmailInput, data.getRandomWords(45));
+        cy.get(emailHelper).should("contain", max120CharsMsg);
+        inputText(stakeholderEmailInput, data.getRandomWord(10));
+        cy.get(emailHelper).should("contain", invalidEmailMsg);
+
+        // Name constraints
+        inputText(stakeholderNameInput, data.getRandomWord(2));
+        cy.get(displayNameHelper).should("contain", minCharsMsg);
+        inputText(stakeholderNameInput, data.getRandomWords(90));
+        cy.get(displayNameHelper).should("contain", max250CharsMsg);
+
+        // Validate the create button is enabled with valid inputs
+        inputText(stakeholderEmailInput, data.getEmail());
+        inputText(stakeholderNameInput, data.getFullName());
+        cy.get(commonView.submitButton).should("not.be.disabled");
+
+        // Close the form
+        cy.get(commonView.cancelButton).click();
+    });
+
+    it("Stakholder button validations", function () {
+        // Navigate to stakeholder tab and click create new button
+        clickByText(navMenu, controls);
+        clickByText(navTab, stakeholders);
+        clickByText(button, createNewButton);
+
+        // Cancel creating new stakeholder
+        cy.get(commonView.cancelButton).click();
+        cy.wait(100);
+
+        clickByText(button, createNewButton);
+
+        // Close create stakeholder form
+        cy.get(commonView.closeButton).click();
+        cy.wait(100);
+
+        // Assert that stakholder tab is opened
+        cy.contains(button, createNewButton).should("exist");
+    });
+
+    it("Stakeholder unique constraint validation", function () {
+        // Create a new stakeholder
+        stakeholder.create();
+        cy.wait("@postStakeholder");
+        exists(stakeholder.email);
+
+        // Navigate to stakeholder tab and click create new button
+        clickByText(button, createNewButton);
+
+        // Check email duplication
+        inputText(stakeholderEmailInput, stakeholder.email);
+        inputText(stakeholderNameInput, data.getFullName());
+        submitForm();
+        cy.get(commonView.duplicateNameWarning).should("contain.text", duplicateErrMsg);
+
+        // Delete created stakeholder
+        cy.get(commonView.closeButton).click();
+        stakeholder.delete();
+        cy.wait("@deleteStakeholder");
+        notExists(stakeholder.email);
+    });
```

Comment:
Do we have any unique value check for dispaly name?
I can see that stakeholder can be created with same display name. and when I go to stakeholder group and try to select stakeholder member then I can see same display name twice. 
So IMO, we should have unique constraint for that also.
I have raised issue - https://issues.redhat.com/browse/TACKLE-188
Please check and add this condition in automation test case